ci-operator/config/heketi/heketi/heketi-heketi-master.yaml

```diff
@@ -0,0 +1,29 @@
+
+canonical_go_repository: github.com/heketi/heketi
+
+base_images:
+  os:
+    cluster: https://api.ci.openshift.org
+    name: centos
+    namespace: openshift
+    tag: '7'
+
+build_root:
+  project_image:
+    dockerfile_path: extras/openshift-ci/build-image/Dockerfile
+
+binary_build_commands: make
+
+tests:
+- as: simple
```

Comment:
Unless you want users to `/test simple` I might suggest a different name here that says more about what the job is doing

Reply:
In heketi currently that's the equvialent of make tests which runs code quality checks and unit tests all as a group. I hesitated to call it unit because it does more than just unit tests. If the standard in openshift-ci is to to use unit for "unit tests and more", I'll happily use that term as it is more common for other projects.